go_pprof_scraper/datadog_checks/go_pprof_scraper/data/conf.yaml.example

```diff
@@ -69,13 +69,17 @@ instances:
     #
     service: <SERVICE>
 
-    ## @param min_collection_interval - number - optional - default: 60
-    ## How frequently profiles are collected.
-    ##
+    ## @param min_collection_interval - number - optional - default: 1
```

Comment:
Despite what it says here, the default if I leave out this parameter is 15 seconds. I'm guessing it's set somewhere else before the check is started? I'd like the default to actually be 1, but as long as these docs are clear and the sample value below is 1, that's okay with me.

Reply:
Yup by default the `min_collection_interval` is [15 seconds](https://docs.datadoghq.com/developers/write_agent_check/?tab=agentv6v7#collection-interval). Just note that having the value be 1 will not _guarantee_ every second:

> If the min_collection_interval is set to 30, it does not mean that the metric is collected every 30 seconds, but rather that it could be collected as often as every 30 seconds. The collector tries to run the check every 30 seconds but the check might need to wait in line, depending on how many integrations are enabled on the same Agent. Also if the check method takes more than 30 seconds to finish, the Agent skips execution until the next interval.

Reply:
Got it, thanks! This integration runs for a long time (60 seconds by default), so my intention is just that once it's done, it starts again as soon as possible. If it's delayed by a second or two, that's okay. Even 15 seconds is okay with me, as long as the user knows it's possible to make it happen more frequently :)